### PR TITLE
Update Read me File

### PR DESCRIPTION
**Pull request by IfzaRasool**
In this PR:

Update Readme File with all Project and Group Member Details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,94 @@
-# medibook-ui
-User interface for the Medi Book health appointment platform
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Book an appointment Front-end
+
+The Project we are going to build is for Final Capstone Project. Doctor Appointment frontend is react frontend application for scheduling doctor appointments.Users need to sign up or sign in to access application.
+
```
Comment (deyemiobaa) [inline README.md]:
```suggestion
This project is our Final Capstone Project. It's a Doctor appointment scheduling app. Built with a React frontend and Rails backend. Users need to sign up or sign in to access the application.

```

**Review by deyemiobaa — APPROVED** [review]
## APPROVED ✅✅

Thank you Ifza. I made some edits other than that this is ready to go.

File: README.md (hunk 1 above)
Comment (deyemiobaa) [reply README.md]: Also, let's add a link to the backend in the readme

Comment (IfzaRasool) [reply README.md]: Yes I already Added

**Pull request merged**